src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline class to represent tasks with deadlines.
+ */
+public class Deadline extends Task {
+    protected String deadline;
+    protected LocalDate time;
```

Comment:
I think it may be better to name the variable as `date`, in case you want to use `LocalTime` or `LocalDateTime` in your future commits.